### DISCUSSION TITLE
socket hang up Error axios Post request

### DISCUSSION THREAD

**Discussion by satishdhavan** [axios/axios#5722] [Q&A]
Hello All,

we are using axios version 1.3.4 in our project to make post request to server and frequently getting "Socket Hangup" Error  immediately after the call, the response object doesn't populate with either response code or data.

const axios = require('axios');
const bossCossClient = axios.create({
  httpsAgent: new https.Agent({
    keepAlive: true
  })
});

Could you let us know if there is an easy way to fix this error without having to retry the request

**Comment by danielsharvey** [comment]
This is probably not related, but I've seen this error with axios as a result of a proxy misconfiguration - see #5741.

**Comment by jasonsaayman** [comment]
Closing check the mentioned issue
